### PR TITLE
D8NID-154 health conditions a-z

### PR DESCRIPTION
**Pull request by neilblair**
Re-order blocks so that facet drop-downs appear above A to Z.
Change name of body location facet to body part.

### CODE REVIEW

File: config/sync/views.view.health_conditions.yml [hunk 1]
```diff
@@ -295,6 +297,10 @@ display:
       display_extenders: {  }
       path: services/health-conditions-a-z
       exposed_block: true
+      cache:
+        type: none
```
Comment (omahm) [inline config/sync/views.view.health_conditions.yml:301]: Shouldn't we have caching by parameter/tag?

**Review by omahm — COMMENTED** [review]
Looks good but I think we should be caching

File: config/sync/views.view.health_conditions.yml (hunk 1 above)
Comment (neilblair) [reply config/sync/views.view.health_conditions.yml:301]: Yeah not sure why this is set to none - wasn't something I altered in the view.  Will take a look

**Review by johangant — COMMENTED** [review]
Looks fine to me, thanks. +1 for the addition of views caching with search API tags.

**Review by omahm — APPROVED** [review]
LGTM 👍

**Pull request merged**